### PR TITLE
[quest] MODIFIED: Druid's "Sunfire" rune locations

### LINKED ISSUE #5446: [quest] 'Sunfire' Druid Rune Fake Quest Missing From QuestDB
## Description

The SoD Druid fake rune quest 'Sunfire' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/guide/season-of-discovery/druid-class-runes-discovery-location
https://www.wowhead.com/classic/spell=414684/sunfire
https://www.wowhead.com/classic/spell=416044/engrave-gloves-sunfire

## Screenshots

N/a

## Questie & WoW version

v9.2.14
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5446 
Linked To #5443 

## Proposed changes

- ADDED: 'Sunfire' Teldrassil Druid Fake Rune Quest is now in the QuestDB, and should be accessible to users.

### CODE REVIEW

File: Database/Corrections/SeasonOfDiscovery.lua [hunk 1]
```diff
@@ -284,6 +285,7 @@ local questsToBlacklistBySoDPhase = {
         [90145] = true, -- Hiding Druid Lacerate Darkshore for now as there are too many icons
         [90146] = true, -- Hiding Druid Mangle Mulgore for now as there are too many icons
         [90147] = true, -- Hiding Paladin Hand of Reckoning Westfall for now as there are too many icons
+        [90148] = true, -- Hiding Druid Sunfire for now as there are too many icons
```
Comment (BreakBB) [inline Database/Corrections/SeasonOfDiscovery.lua:288]: Instead of hiding this, we should cleanup the Mulgore spawns of the "Lunar Stone" NPC (207577)

Comment (Sommos) [reply Database/Corrections/SeasonOfDiscovery.lua]: Fixed in: https://github.com/Questie/Questie/pull/5457/commits/3cdd3bb1b35173d83f9b608c62aa760dd301d2cb

File: Database/Corrections/sodNPCFixes.lua [hunk 2]
```diff
@@ -86,6 +86,11 @@ function SeasonOfDiscovery:LoadNPCs()
                 [zoneIDs.MULGORE] = {{30.6, 61.2}},
             },
         },
+        [207577] = { -- Lunar Stone
+            [npcKeys.spawns] = {
+                [zoneIDs.MULGORE] = {{35.2, 69.6}},
+            },
```
Comment (BreakBB) [inline Database/Corrections/sodNPCFixes.lua:92]:
```suggestion
            [npcKeys.spawns] = {
                [zoneIDs.TELDRASSIL] = {{52.8, 79.8}},
                [zoneIDs.MULGORE] = {{35.2, 69.6}},
            },
```

We want to keep the TELDRASSIL location.

I am fine though by reducing these spawns to a single one per zone even thought there are three. Tooltips will show nevertheless 👍🏻

Comment (Sommos) [reply Database/Corrections/sodNPCFixes.lua:92]: Yeah, I think it's best to have them set to a single spawn point as they are so close to each other in both Teldrassil & Mulgore that they just overlap and it looks messy.

**Pull request merged**